lib/vault.py

```diff
@@ -223,7 +245,7 @@ def configure(self):
                                             region = aws_creds.get("aws_region", "us-east-1"))
```

Comment:
The call to `aws/config/root` can be eliminated, because like the AWS auth the SDK will read the instance's IAM role